pyproject.toml

```diff
@@ -151,7 +151,8 @@ markers = [
     "fastapi",
     "chalice",
     "flask",
-    "starlite"
+    "starlite",
+    "flaky"
```

Comment:
Is it OK to add the `flaky` marker in that PR?
Or should it go into a separate PR?

Reply:
yes 😊